### PR TITLE
Add Noda Time project data

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: _data/projects/nodatime.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Noda Time
+desc: A better date/time API for .NET
+site: https://github.com/nodatime/nodatime
+tags:
+- C#
```
Comment (jskeet) [inline _data/projects/nodatime.yml]: Would .NET be useful here as well? It's not clear whether the .NET and C# tags are effectively equivalent...

Comment (shiftkey) [reply _data/projects/nodatime.yml]: We're mostly figuring out our tag set over in #678 but this is worth doing for the moment.

Comment (jskeet) [reply _data/projects/nodatime.yml]: Righto - added in another commit.

**Comment by shiftkey** [comment]
@jskeet thanks!

**Pull request merged**